package-lock.json

```diff
@@ -3751,26 +3751,6 @@
         }
       }
     },
-    "@testing-library/user-event": {
```

Comment:
Why is package-lock.json changing without a change to package.json?

Reply:
I was wondering the same thing... Do you want me to remove this change from the PR?